config/initializers/pagy.rb

```diff
@@ -1,12 +1,8 @@
-# frozen_string_literal: true
-
-# Array extra: Paginate arrays efficiently, avoiding expensive array-wrapping and without overriding
-# See https://ddnexus.github.io/pagy/docs/extras/array
 require 'pagy/extras/array'
-
-# Overflow extra: Allow for easy handling of overflowing pages
-# See https://ddnexus.github.io/pagy/docs/extras/overflow
 require 'pagy/extras/overflow'
-Pagy::DEFAULT[:overflow] = :empty_page # default  (other options: :last_page and :exception)
+require 'pagy/extras/size'
 
+Pagy::DEFAULT[:overflow] = :empty_page # default  (other options: :last_page and :exception)
+Pagy::DEFAULT[:size] = [1, 1, 1, 1].freeze
+Pagy::DEFAULT[:limit] = 10
```

Comment:
10 items per page seems like a decent number, the cards are quiet tall and any more is a load of scrolling.